router/router.go

```diff
@@ -51,6 +51,17 @@ type Router interface {
 	Logger() stdlog.StdLog
 }
 
+// An DynamicRouter is a router that allows for dynamic Realm adding.
+type DynamicRouter interface {
+	Router
+
+	// AddRealm will append a realm to this router
+	AddRealm(*RealmConfig) error
+
+	// RemoveRealm will attempt to remove a realm from this router
+	RemoveRealm(wamp.URI)
+}
```

Comment:
Thinking we should just add `AddRealm` and `RemoveRealm` to `Router`.  I do not think a new interface is needed.

Reply:
Please combine.  Weighing the risk of breaking bc versus what looks right here, I choose to go with what results in "better" code.  Submit change, and I I'll do a final review and commit.  Thank you.